src/App/PropertyExpressionEngine.cpp

```diff
@@ -1079,3 +1080,47 @@ void PropertyExpressionEngine::onRelabeledDocument(const App::Document &doc)
             e.second.expression->visit(v);
     }
 }
+
+void PropertyExpressionEngine::getLinksTo(std::vector<App::ObjectIdentifier>& identifiers,
+                                          App::DocumentObject* obj,
+                                          const char* subname,
+                                          bool all) const
+{
+    Expression::DepOption option =
+        all ? Expression::DepOption::DepAll : Expression::DepOption::DepNormal;
+
+    App::SubObjectT objT(obj, subname);
+    auto sobj = objT.getSubObject();
+    auto subElement = objT.getOldElementName();
+
+    for (auto& v : expressions) {
```

Comment:
All of these pairs and single-char variable names are very difficult to parse: I suggest unpacking the pairs using structured bindings so that instead of `v.first` and `v.second` (for example) you can use `expressionID` and `expressionInfo`.

Reply:
Fixed in #15501